PyPDF2/generic.py

```diff
@@ -578,6 +578,29 @@ def writeToStream(self, stream, encryption_key):
 
     @staticmethod
     def readFromStream(stream, pdf):
+        def getNextObjPos(p, p1, remGens, pdf):
+            l = pdf.xref[remGens[0]]
+            for o in l:
+                if p1 > l[o] and p < l[o]:
+                    p1 = l[o]
+            if len(remGens) == 1:
+                return p1
+            else:
+                return getNextObjPos(p, p1, remGens[1:], pdf)
+
+        def readUnsizedFromSteam(stream, pdf):
+            # we are just pointing at beginning of the stream
+            eon = getNextObjPos(stream.tell(), 2**32, [g for g in pdf.xref], pdf) - 1
```

Comment:
what does "eon" mean?

Reply:
end before next object : we have to find when to stop looking for endstream